torch/testing/_internal/common_methods_invocations.py

```diff
@@ -11053,11 +11053,15 @@ def error_inputs_mean(op_info, device, **kwargs):
                        DecorateInfo(unittest.skip("Skipped!"), 'TestUnaryUfuncs', 'test_reference_numerics_extremal',
                                     device_type='cpu',
                                     dtypes=[torch.cfloat, torch.cdouble], active_if=IS_MACOS),
+                       #  AssertionError: Tensor-likes are not close!
+                       DecorateInfo(unittest.expectedFailure, 'TestUnaryUfuncs', 'test_reference_numerics_large',
+                                    device_type='cuda',
+                                    dtypes=(torch.chalf,)),
```

Comment:
Ref: #78458 (Failue: https://hud.pytorch.org/pytorch/pytorch/commit/5fbec86faef07d66ab696bc4c4edbaf6259a2189).

Thanks!

Reply:
Since it only failed on Windows, you might have to set `active_if=WINDOWS` or something like this in `DecorateInfo` so that the expectedFailure is applied only for Windows CI.

Reply:
Ah, yes! Fixed. Thanks!